src/client/index.tsx

```diff
@@ -17,8 +17,11 @@ const CONTAINER = document.getElementById('react-root');
 
 // Underline the current item in the top-nav. Plain JS. Uses tachyons classes.
 document.querySelectorAll('[data-hl-nav]').forEach(node => {
+  let HTMLEle:HTMLElement = node as HTMLElement;
+  HTMLEle.style
```

Comment:
What is this line?

Reply:
querySelectorAll returns an Element and NOT HTMLELement. If you want to use .style, then you have to convert  it to HTMLElement

Reply:
But the line is just `HTMLEle.style` -- it doesn't do anything